app/renderer/js/utils/system-util.ts

```diff
@@ -28,22 +33,23 @@ class SystemUtil {
 	}
 
 	getOS() {
-		if (os.platform() === 'darwin') {
+		const platform: string = os.platform();
```

Comment:
I remove the `string` declaration there. The function return a type `NodeJS.Platform` which we want to use instead. Genral rule is you shouldn't annotate type that are implicitly annotated.

For example:
```typescript
// Do
const a: string;
a = 'aa';

// Don't
const a: string = 'aa';
```

Reply:
Got it! Will keep in mind from now on.